precise/iaf_psc_exp_ps_lossless.cpp

```diff
@@ -414,18 +414,20 @@ nest::iaf_psc_exp_ps_lossless::update( const Time& origin,
       {
         // time is measured backward: inverse order in difference
         const double ministep = last_offset - ev_offset;
+        assert( ministep >= 0.0 );
+        if ( ministep > 0.0 )
+        {
+          propagate_( ministep );
 
-        propagate_( ministep );
-
-        // check for threshold crossing during ministep
-        // this must be done before adding the input, since
-        // interpolation requires continuity
-
-        const double spike_time_max = is_spike_( ministep );
+          // check for threshold crossing during ministep
+          // this must be done before adding the input, since
+          // interpolation requires continuity
+          const double spike_time_max = is_spike_( ministep );
 
-        if ( not numerics::is_nan( spike_time_max ) )
-        {
-          emit_spike_( origin, lag, V_.h_ms_ - last_offset, spike_time_max );
+          if ( not numerics::is_nan( spike_time_max ) )
+          {
+            emit_spike_( origin, lag, V_.h_ms_ - last_offset, spike_time_max );
+          }
         }
 
```

Comment:
@heplesser  thanks!
If I see it correctly, the issue of a `ministep == 0` is handled slightly differently in `iaf_psc_exp_ps.cpp`, where this is done by checking that `dt != 0` in `propagate_( dt )` and `emit_spike( ... , dt )`.
I think it does the same, but just wanted to mention this in case you didn't do it differently on purpose

Reply:
@ChristianKeup Ah, I overlooked that. I will update my code.